eclipse-collections-testutils/src/main/java/org/eclipse/collections/impl/test/Verify.java

```diff
@@ -240,14 +234,6 @@ public static void assertIterableEmpty(String iterableName, Iterable<?> iterable
         }
     }
 
-    /**
-     * Assert that the given object is an instanceof expectedClassType.
-     */
-    public static void assertInstanceOf(Class<?> expectedClassType, Object actualObject)
-    {
-        Verify.assertInstanceOf(actualObject.getClass().getName(), expectedClassType, actualObject);
-    }
```

Comment:
Why delete this? You didn't delete the other override of the same method so it's not like it's unsupported.